components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -467,13 +467,26 @@ export const IconButtonContainer = styled('div')<IconButtonContainerProps>`
   font-family: ${p => p.theme.fontFamily.heading};
   font-size: 13px;
   font-weight: 600;
-  color: rgba(255,255,255,0.8);
+  color: rgba(var(--override-readability-color-rgb, 255, 255, 255), 0.8);
   margin-right: ${p => p.textDirection === 'ltr' && '8px'};
   margin-left: ${p => p.textDirection === 'rtl' && '8px'};
-  border-right: ${p => p.textDirection === 'ltr' && '1px solid rgba(255, 255, 255, 0.6)'};
-  border-left: ${p => p.textDirection === 'rtl' && '1px solid rgba(255, 255, 255, 0.6)'};
+  border-right: ${p => p.textDirection === 'ltr' && 'rgba(var(--override-readability-color-rgb, 255, 255, 255), 0.6)'};
+  border-left: ${p => p.textDirection === 'rtl' && 'rgba(var(--override-readability-color-rgb, 255, 255, 255), 0.6)'};
 
   &:hover {
-    color: #ffffff;
+    color: ${p => p.color};
+  }
+`
+
+export const OverrideReadabilityColor = createGlobalStyle<{override: boolean}>`
+  body {
```

Comment:
Optional nit: 
```suggestion
  :root {
```

I think we set most other CSS variables on the root, so we can do that with yours too

Reply:
Yep that's better! thanks.